Update Twilio/EDI connector release-note for U9 (#8920)
ballerina-platform/ballerina-dev-website: downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

#### `twilio` package

> - Enhanced integration with Twilio services (messaging, voice calls, media services, and more) offered through the Twilio Basic REST API version 2010-04-01. 

Reviewer: ```suggestion
- Enhanced integration with Twilio services (messaging, voice calls, media services, etc.) offered via the Twilio Basic REST API version 2010-04-01. 
```

> - Enhanced integration with Twilio services, now fully supporting all features offered through the Twilio Basic REST API version 2010-04-01. This comprehensive update includes capabilities across messaging, voice calls, media services, and more, ensuring full utilization of Twilio's extensive functionality.

Reviewer 2: ```suggestion
- Enhanced integration with Twilio services (messaging, voice calls, media services, and more) offered through the Twilio Basic REST API version 2010-04-01. 
```
